src/app/parcels/parcelsTable/styles.ts

```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
 
 export const parcelTableColumnStyleOptions = {
     iconsColumn: {
-        width: "4rem",
+        maxWidth: "4rem",
+        minWidth: "0.7rem",
+        grow: 0.5,
     },
     fullName: {
-        grow: 1.3,
-        minWidth: "8rem",
+        grow: 1.2,
+        minWidth: "6.5rem",
     },
     familyCategory: {
         hide: 550,
-        grow: 0.8,
+        grow: 0.7,
         minWidth: "6rem",
     },
     addressPostcode: {
         hide: 800,
-        grow: 0.8,
-        minWidth: "6rem",
+        grow: 0.6,
+        minWidth: "5rem",
+        maxWidth: "6rem",
     },
     deliveryCollection: {
-        grow: 0.8,
+        grow: 0.6,
         minWidth: "6rem",
     },
     packingDate: {
         hide: 800,
-        grow: 0.8,
-        minWidth: "6rem",
+        grow: 0.7,
+        minWidth: "7rem",
     },
     packingSlot: {
         hide: 800,
-        grow: 0.8,
-        minWidth: "5rem",
+        grow: 0.5,
+        minWidth: "4rem",
     },
     lastStatus: {
         grow: 3,
         maxWidth: "30rem",
+        minWidth: "10rem",
     },
 };
```

Comment:
gave last status a min width to make it larger on a smaller screen, and adjusted the sizing of the other columns to be proportionally smaller